suite-native/module-authorize-device/src/screens/connect/ConnectAndUnlockDeviceScreen.tsx

```diff
@@ -25,6 +29,10 @@ export const ConnectAndUnlockDeviceScreen = ({
     RootStackParamList
 >) => {
     const isBluetoothEnabled = useFeatureFlag(FeatureFlag.IsBluetoothEnabled);
+    const isBluetoothSupportedByDevice = useSelector(selectIsBluetoothSupportedByDevice);
+    const isBluetoothButtonVisible =
+        // if the model does not support bluetooth, we should not show the button,
+        isBluetoothSupportedByDevice && isBluetoothEnabled;
```

Comment:
Would swap the order here:
```suggestion
        isBluetoothEnabled && isBluetoothSupportedByDevice;
```

Reply:
https://github.com/trezor/trezor-suite/pull/21761/commits/daf8e413eb77ad1fe2f819a77396173187561ae3 done ☑️